### PR TITLE
Link to fingerprinting grammar

### PR DESCRIPTION
**Pull request by jjbayer**
Fingerprint rules do not support the full scope of Discover queries. Update docs to make this more clear.

### CODE REVIEW

File: src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx [hunk 1]
```diff
@@ -15,7 +15,7 @@ These rules can be configured on a per-project basis in **[Project] > Settings >
 1. Identify the match logic for grouping issues together.
 1. Set the match logic and the fingerprint for it.
 
-The syntax follows the [syntax](/product/sentry-basics/search/#syntax) from **Discover** queries. If you want to negate the match, prefix the expression with an exclamation mark (`!`).
+The syntax for fingerprint rules is similar to that of [**Discover** queries](/product/sentry-basics/search/#syntax). If you want to negate the match, prefix the expression with an exclamation mark (`!`). The full grammar is defined [here](https://github.com/getsentry/sentry/blob/c5b84a393365a833348dd7fd9378d34c353ca6ca/src/sentry/grouping/fingerprinting.py#L17-L55).
```
Comment (lizokm) [inline src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx:18]:
```suggestion
The syntax for fingerprint rules is similar to syntax in [**Discover** queries](/product/sentry-basics/search/#syntax). To negate a match, add an exclamation mark (`!`) at the beginning of the expression. See the full list of grammar [here](https://github.com/getsentry/sentry/blob/c5b84a393365a833348dd7fd9378d34c353ca6ca/src/sentry/grouping/fingerprinting.py#L17-L55).
```

**Review by lizokm — APPROVED** [review]
Just a few small language tweaks, otherwise this looks great.

File: src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx (hunk 1 above)
Comment (jjbayer) [inline src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx:18]:
```suggestion
The syntax for fingerprint rules is similar to syntax in [**Discover** queries](/product/sentry-basics/search/#syntax). To negate a match, add an exclamation mark (`!`) at the beginning of the expression. See the full grammar [here](https://github.com/getsentry/sentry/blob/c5b84a393365a833348dd7fd9378d34c353ca6ca/src/sentry/grouping/fingerprinting.py#L17-L55).
```

Comment (jjbayer) [reply src/docs/product/data-management-settings/event-grouping/fingerprint-rules.mdx]: Thanks! `list of grammar` did not sound right to me, so I changed it to "See the full grammar here".

**Pull request merged**